packages/sdk/src/types.ts

```diff
@@ -792,7 +792,7 @@ export interface ChainArData {
 export interface Policy {
     validTo: string; // "YYYYMM"
     createdAt: string; // "YYYYMM"
-    revealedAttributes: Record<AttributeKey, string>;
+    revealedAttributes: Partial<Record<AttributeKey, string>>;
```

Comment:
I changed it because my IDE was complaining 🙂

Your test checks if `one` and `two` satisfies _either_ `First` or `Second`, so it passes because it satisfies `Second`. If you remove `| Second` it no longer works.

Reply:
You're right, sorry for that 😜. The previous version is even incorrect, so I'll eat my own hat 🎩. Good find!